apps/tangle-dapp/components/BnChartTooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import { Typography } from '@webb-tools/webb-ui-components/typography/Typography';
+import BN from 'bn.js';
```

Comment:
I'm not sure why we need to use `BN` with `bn.js` lib here, `polkadotjs` already provides this